{{cookiecutter.app_name}}/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/nimblehq/{{cookiecutter.app_name}}
```

Comment:
To keep the template code away from the `gin-templates` app codebase, I recommend to have a directory named `template` and place all code for the app template in there.

For now, there would be only `go.mod`.

With this, please check if we can avoid having the directory `{{cookiecutter.app_name}}` by creating this by ourselves and then adding all files from the template directory. It's not great to have a directory named with braces 🥴

Reply:
About an hour research, cookiecutter doesn't support what you mentioned atm. I opened an issue to ask them about what we need and waiting for their reply now.

We can do like this
```
.
├── .github/
├── cmd/
├── template/
    ├── {{cookiecutter.app_name}
        ├── go.mod
    ├── cookiecutter.json
├── main.go
```

Reply:
Too bad. Then that's a tradeoff we would need to consider when organizing the codebase